lib/ci-stack.ts

```diff
@@ -80,6 +90,33 @@ export class CIStack extends Stack {
       oidcCredArn: importedOidcConfigValuesSecretBucketValue.toString(),
       useSsl,
       runWithOidc,
+    },
+    {
+      region: this.region.toString(),
```

Comment:
region does not need to be passed in, it can be extracted from `stack.region`

Reply:
The method is not a part of the constructor so need to pass that as a parameter https://github.com/opensearch-project/opensearch-ci/pull/39/files#diff-90fd20658ffdeb68fbac6a146498601102a44c741138481202ff494189e69c30R50
Should I pass stack as a param then and then extract region from it?

Reply:
Sure, but do that at the last possible minute in the code so you aren't passing that parameter around when it is unused.

Reply:
Done